app/routes/projects-list.js

```diff
@@ -1,6 +1,14 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  currentUser: Ember.inject.service(),
+
+  beforeModel() {
+    this._super(...arguments);
+    let user = this.get('currentUser.user');
+    user.get('skill');
+  },
```

Comment:
What made you think it was necessary? Removing this seems to be ok.. but curious what your concern was and why it was added.